eng/Versions.props

```diff
@@ -70,4 +70,8 @@
       https://dotnetfeed.blob.core.windows.net/orchestrated-release-2-1/20180725-02/final/index.json
     </RestoreSources>
   </PropertyGroup>
+  <!--Package names-->
+  <PropertyGroup>
```

Comment:
This got created when I ran 'darc add'. Wasn't sure if this is needed.

Reply:
This should not be needed. Can you remove?

cc @mmitche to confirm though.

Reply:
Will wait for @mmitche to confirm

Reply:
It's not needed.  I plan on removing it.